styles/components/_button.scss

```diff
@@ -53,7 +53,7 @@
     border: 1px solid $brand-gray;
   }
 
-  &.-primary { @include btn-variant($brand-sidebar-lt); }
+  &.-primary { @include btn-variant($brand-primary); }
```

Comment:
I think there's a huge misunderstanding. Last time, we talked about changing the primary button to `$brand-sidebar-lt`. Why the change again? :/